jitbuilder/optimizer/JBOptimizer.cpp

```diff
@@ -117,11 +117,12 @@ static const OptimizationStrategy JBwarmStrategyOpts[] =
    { OMR::treeSimplification                                                       },
    { OMR::trivialDeadTreeRemoval,                    OMR::IfEnabled                },
 
-   { OMR::inductionVariableAnalysis,                 OMR::IfLoops                  },
-   { OMR::loopCanonicalization,                      OMR::IfLoops                  },
+   { OMR::basicBlockOrdering                         OMR::IfLoops                  }, // clean up block order for loop canonicalization, if it will run
```

Comment:
Thanks.  This needs a ',' after `OMR::basicBlockOrdering` to avoid the TravisCI break.